### DISCUSSION TITLE
Airfoil surface area

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1868] [Q&A]
How to calculate the airfoil surface area or where can find the airfoil surface area data?

**Comment by jjonkman** [comment]
Dear @sjihf12,

Are you referring to the circumference around a given 2D airfoil?  If you have an AeroDyn airfoil coordinate data file for the given airfoil, you could certainly sum the distances between each consecutive point.

Best regards,

**Reply by sjihf12** [reply]
I'm sorry, I meant 2D airfoil surface area.

**Comment by sjihf12** [comment]
Dear Professor
&nbsp;If I change the shape parameters of floater, How do I calculate the "PLATFORM ADDITIONAL STIFFNESS AND DAMPING" in HydroDyn.dat&nbsp;






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "OpenFAST/openfast"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月12日(星期天) 凌晨2:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [OpenFAST/openfast] Airfoil surface area (Discussion #1868)





 
Dear @sjihf12,
 
Are you referring to the circumference around a given 2D airfoil?  If you have an AeroDyn airfoil coordinate data file for the given airfoil, you could certainly sum the distances between each consecutive point.
 
Best regards,

**Reply by jjonkman** [reply]
Dear @sjihf12,

The platform additional stiffness and damping in HydroDyn are meant to be used to capture stiffness and damping that are not otherwise directly considered in other parts of the model and are typically calibrated to known data.  In many cases, these are zeroed.  See the HydroDyn documentation for more information: https://openfast.readthedocs.io/en/main/source/user/hydrodyn/input_files.html#platform-additional-stiffness-and-damping and https://openfast.readthedocs.io/en/main/source/user/hydrodyn/modeling_considerations.html#platform-additional-stiffness-and-damping.

Best regards,

**Comment by lt72884** [comment]
For a paper i had published by IEEE in November, i had to come up with a way to find the area of the air foil. So i used an equation then integrated. It worked for my application, maybe it will work for yours? If not, at least you have the equations now :)

![NACA 4 digit equation](https://github.com/OpenFAST/openfast/assets/52751207/fd110112-79b6-4efd-a60a-176efbe7ce1f)

![image](https://github.com/OpenFAST/openfast/assets/52751207/20d0ec48-130d-4f10-bab1-f1daee1509fb)

![image](https://github.com/OpenFAST/openfast/assets/52751207/562bfa79-2c16-4e14-9aaf-fac7fc9519e9)
